scripts/live-migration-times.d

```diff
@@ -138,6 +142,10 @@ dtrace:::END
 		printf("%-16s %29d\n", this->rpush_post, this->d_rpush_post / 1000);
 		this->total += this->d_rpush_post;
 	}
+	if (this->d_td != 0) {
+		printf("%-15s %30d\n", this->td, this->d_td / 1000);
```

Comment:
nit: same width specifiers as the surrounding prints? (If the output aligns properly, then no worries; I just remember having to mess around with this to get all the headers to line up right when I added the post-pause phase.)

Reply:
will fix this (and my own D script nits https://github.com/oxidecomputer/propolis/pull/359#discussion_r1152605234) in a follow on PR